src/formatDate.js

```diff
@@ -8,7 +8,36 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let resultDate = '';
+  const objectDateFormat = {};
+  const dateSplitted = date.split(fromFormat[fromFormat.length - 1]);
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    const less = dateSplitted[i] < 30;
+    const more = dateSplitted[i] >= 30;
```

Comment:
The variables 'less' and 'more' are not descriptive of their purpose and can be confusing. Consider renaming them to reflect that they are used to determine the century when converting from 'YY' to 'YYYY'. For example, 'is20thCentury' and 'is21stCentury' would be more descriptive.